akka-docs/rst/project/migration-guide-persistence-experimental-2.3.x-2.4.x.rst

```diff
@@ -180,6 +180,27 @@ Read more about at-least-once delivery in the :ref:`documentation for Scala <at-
 :ref:`documentation for Java <at-least-once-delivery-java>`.  
 
 
+Added support for multiple persistent plugin configurations
+===========================================================
+
+By default, persistent actor will use journal and snapshot store plugins
+configured in the following sections of the ``reference.conf`` configuration resource ::
+
+akka.persistence.journal.plugin = "place here absolute path to the journal plugin config entry"
+akka.persistence.snapshot-store.plugin = "place here absolute path to the snapshot store plugin config entry"
+
+And when persistent actor overrides ``journalPluginId`` and ``snapshotPluginId`` methods ::
+
+    class MarketState extends PersistentActor {
+    
+      override def persistenceId = "EUR/USD"
+      
+      // Absolute path to the journal configuration entry in the `reference.conf`.
+      override def journalPluginId = "akka.persistence.chronicle.journal"
+      
+      // Absolute path to the snapshot store configuration entry in the `reference.conf`.
+      override def snapshotPluginId = "akka.persistence.chronicle.snapshot-store"
+      
+    }
 
-
-   
\ No newline at end of file
+Then the actor will be serviced by these specific persistence plugins. 
```

Comment:
slightly strange to continue sentence after the sample.

"When persistent actor overrides ``journalPluginId`` and ``snapshotPluginId`` methods the actor will be serviced by these specific persistence plugins instead of the defaults."